iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -693,6 +694,13 @@ - (void)updateTrack:(MWMTrackID)trackId
         return;
     }
 
+    auto const currentColor = track->GetColor(0);
+    auto const newColor = [self getColorFromUIColor:color];
+
+    if (newColor != currentColor) {
+        track->SetColor(newColor);
+    }
```

Comment:
1. No {} for single line ifs or loops.
2. Please set 2 spaces indentation instead of 4.
```suggestion
    if (newColor != currentColor)
      track->SetColor(newColor);
```